src/datagrid/DataGrid.tsx

```diff
@@ -393,20 +404,19 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
         let rows = this.updateRowIDs(allRows);
         const columns = this.updateColumnIDs(this.setColumnVisibility(allColumns));
         columns.forEach(function(col) {
```

Comment:
why to write native functions,  can we use the ()=>  instead.

Reply:
done